README.md

```diff
@@ -13,27 +13,28 @@ It's nice to have you here! This project is the web app created by the PodCodar
   <img src="./docs/images/linux.svg" height="30" />
 </div>
 
-### Start Project
+### Getting Started
+To work on this project, you can use either `yarn` or `bun` as your package manager.
 
-To work on this project you have use `yarn` or `bun` as package manager.
+> Personal recommendation: Just use `bun` and be happy.
 
-> Personal suggestion: use `bun` and be happy.
+- Install dependencies using Bun by running the command `bun install`.
 
-- Install dependencies using library bun whit command `bun`.
+- Run `bun decrypt` to decrypt the `.env.gpg` file. (It'll require a password. Ask current team members for it.)
 
-- run `bun decrypt` to decrypt the `.env.gpg` file. (ask the password for the current teammembers)
+> Note: if you're using Windows, you'll need to configure GnuPG before decrypting the `.env.gpg` file. GnuPG can be installed from a package manager —like chocolatey— or it can be downloaded directly at [gpg4win](https://gpg4win.org/download.html)
```

Comment:
I'll make a hard suggestion, but....
```suggestion
> Note: if you're using Windows, you'll need to install a real Operational System, compatible with GNU/Linux - [🐧 Instalando Linux](https://podcodar.notion.site/Instalando-Linux-d78e8dcd9adf429b9a409b974c0d8fe5)
```

Some reasons why it's harder to work on Windows in open-source projects, (e.g. any non-Microsoft tools):
- Windows manages environment variables poorly and conflicts with Linux
- No package manager by default
- No `sudo` and a really bad PowerShell user experience
- On Windows, line-endings are terminated with a combination of a carriage return `(ASCII 0x0d or \r)` and a newline `(\n)`, also referred to as `CR/LF`
- The production environment is running Linux

For those reasons, you will probably end up installing `WSL` which is a second Operational System to run Linux inside Windows - because they give up on building a good OS for developers - so, why not just use Linux?

References:
- https://portal.perforce.com/s/article/3096#:~:text=On%20Windows%2C%20line%2Dendings%20are,referred%20to%20as%20CR%2FLF.
- https://learn.microsoft.com/en-us/windows/wsl/troubleshooting
- https://forums.docker.com/t/issues-with-docker-desktop-and-wsl-2-integration/141865

Reply:
Makes sense to me. The project will only be run by mentors and mentees, right ? So as far as I can see there is no need to cater to the Windows users.
Should I just take this whole section off or should I keep it with a message communicating the preference for running the project on Linux ? Something like:

> Note: We strongly recommend running the project on Linux. If you have Windows as your OS, or some other non-GNU/Linux compatible OS, see [🐧 Instalando Linux](https://podcodar.notion.site/Instalando-Linux-d78e8dcd9adf429b9a409b974c0d8fe5)

What do you think ?

Reply:
Looks good! 👍🏼